src/constants-internal.ts

```diff
@@ -19,6 +19,9 @@ export const DEFAULT_MAX_LENGTH_URL = 7000;
 /** @privateRemarks Source: @deck.gl/carto */
 export const DEFAULT_TILE_RESOLUTION = 0.5;
 
+/** @internal */
+export const REDUCED_QUERIES_TILE_RESOLUTION = 1;
+
```

Comment:
The 'reduced queries' value is really a reference to an old feature flag, we should probably update `DEFAULT_TILE_RESOLUTION` to 1.0 the next time there's a breaking change opportunity. I've been using the `TODO(v0.5)` prefix for that purpose:

```suggestion
/**
 * @todo TODO(v0.5): Update DEFAULT_TILE_RESOLUTION and remove this constant.
 * @internal
 */
export const REDUCED_QUERIES_TILE_RESOLUTION = 1;

```